### PR TITLE
EDITORIAL CLOSING: XMPP Newsletter month about August 2020 

### PR DESCRIPTION
**Pull request by Echolon**
I created the draft for the next release August 2020.

You can make a simple comment or create a merge request to make things easy going for review. If you need help, please ask for help in the Communication-Team MUC: commteam@muc.xmpp.org

Everyone participating, please do not just drop bare links - rather try to make a short and decent formulated description what is the main outcome of your source. You do not need to write formal - be passionate why this motivates you or what you work on! 
You also do not need a command-line tool or any knowledge of git to commit your changes. (Of course, pull requests are highly appreciated)

- If you have write access add your comment as follows: Click the `Files changed` ribbon, then the three dots on the top right, then `Edit file`. Finally add you comment.
- Just paste your prepared text here below or the our channel (commteam@muc.xmpp.org)

This is our planed timeline for this draft:

XMPP Newsletter Timeline

CC: @jcbrand @Nyco @SeveFP

**Note: If you create a merge request, which is highly appreciated, please @mention me once your PR is created!** :pray: 

Status:
- [X] Links collection
- [X] Review
- [ ] Blog post + Tiny Newsletter
- [ ] Social media
- [ ] Translations
- [ ] Archived ([Use pandoc!](https://pandoc.org))

### CODE REVIEW

**Comment by Echolon** [comment]
@goffi-contrib this really goes here: https://www.goffi.org/b/n9nzn3ySF7Lv4pYBpTqYtP/progress-note
if you have sometime in the next month a short summary formulation from your side of course would be highly appreciated!

**Comment by Echolon** [comment]
FYI @mar-v-in, @melvo, https://github.com/xsf/xmpp.org/pull/756/commits/24b769a615f3f320da74e87aaa5e96617478b200

**Comment by Echolon** [comment]
**Open tasks to do** (I am happy for support :smile_cat:)

Add:

_(maybe some are already included, not sure)_

- https://discourse.igniterealtime.org/t/jsxc-web-client-now-available-as-a-plugin-for-openfire/88467
- https://github.com/anurodhp/Monal/releases/tag/Build_iOS_636
- https://github.com/matrix-org/prosody-mod-auth-matrix-user-verification
- https://blog.jabberhead.tk/2020/09/01/pgpainless-0-1-0-released/
- https://tigase.net/tigase-meets-docker/
- https://tigase.net/halcyon-a-look-at-halcyon/
- https://discourse.igniterealtime.org/t/smack-4-4-0-beta1-released/88519
- https://discourse.igniterealtime.org/t/openfire-4-5-3-is-released/88548
- https://discourse.igniterealtime.org/t/client-control-plugin-2-1-5-released/88550
- https://discourse.igniterealtime.org/t/spark-2-9-0-released/88551
- https://discourse.igniterealtime.org/t/jsxc-web-client-now-available-as-a-plugin-for-openfire/88467

**Comment by Echolon** [comment]
@guusdk FYI 8f150eb

**Comment by Echolon** [comment]
@RincewindsHat  FYI 1c28cdc

File: content/posts/newsletter/2020-08-08.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ The iOS and macOS client Monal is organizing translations for the app on [Weblat
 
 The Android client aTalk 2.4.0 has been released. [Changes](https://github.com/cmeng-git/atalk-android/blob/master/aTalk/ReleaseNotes.txt) are mainly internal with several libraries upgrades and compilation fixes. It was followed by version 2.4.1 which fixes an Android Q specific storage issue.
 
+The desktop client [Dino received a NGI0 PET Fund](https://dino.im/blog/2020/08/setting-out-for-calls-and-conferences/) which is is managed by NLnet which further receives financial support from European Commission’s Next Generation Internet programme. This allows the developer to implement video and audio calls into their clients. This is really great news!
+
+[Kaidan 0.6.0 has been released](https://www.kaidan.im/2020/08/20/kaidan-0.6.0/)! The update includes a new offline message queue as well as an improved message correction. There is upstream HTTP file upload implementation now and also further refactoring.
```
Comment (melvo) [inline content/posts/newsletter/2020-08-08.md:45]: Thanks @Echolon! I would adjust the text a bit:

```suggestion
[Kaidan 0.6.0 has been released](https://www.kaidan.im/2020/08/20/kaidan-0.6.0/)! The update includes stability fixes, a new offline message queue and an improved message correction. Furthermore, the upstream HTTP File Upload implementation of its XMPP library [QXmpp](https://github.com/qxmpp-project/qxmpp) is used now.
```

**Comment by Echolon** [comment]
### :hourglass:  **Editorial closing** 

### :rocket: Release by tomorrow night

### :pray: Thanks to everyone of you!  
**COM8, emus, jcbrand, jerome-poisson, jonas', Licaon_Kter, melvo, pmaziere, SeveFP, vanitasvitae, wurstsalat, zash**
### :handshake:   ... okay okay its Corona time :hand: :mask: :hand:

**Pull request merged**